apollo-router/src/services/layers/persisted_queries/manifest_poller.rs

```diff
@@ -795,6 +847,6 @@ mod tests {
         )
         .await
         .unwrap();
-        assert_eq!(manifest_manager.get_operation_body(&id), Some(body))
+        assert_eq!(manifest_manager.get_operation_body(&id, None), Some(body))
```

Comment:
Not specific to this line, let's have a test that exercises the fallback logic of get operation body in this file.
Test:
* PQ for a client.
* PQ for a client doesn't exist falls back to non client specific PQ.
* No PQ at all

Reply:
We do have that in integration tests. You'd like it in the unit tests too?

Reply:
If it can be unit tested then yes. Time to discover a regression via integration test is super high compared to unit test. It's also nice if there is test code close to the functionality.